spec/avram/queryable_spec.cr

```diff
@@ -856,6 +856,95 @@ describe Avram::Queryable do
     end
   end
 
+  describe "#group_count" do
+    context "when no records exist" do
+      it "0 grouped columns returns { [] => 0 }" do
+        query = UserQuery.new
+
+        query.group_count.should eq({ [] of String => 0 })
+      end
+
+      it "1 grouped columns returns { [] => 0 }" do # TODO: What should we do here?
+        query = UserQuery.new.group(&.name)
+
+        query.group_count.should eq({ [] of String => 0 })
+      end
+
+      it "2 grouped columns returns { [] => 0 }" do # TODO: What should we do here?
+        query = UserQuery.new.group(&.name)
+
+        query.group_count.should eq({ [] of String => 0 })
+      end
+    end
+
+    context "when 1 record exists" do
+      before_each do
+        UserFactory.create do |user|
+          user.age(32)
+          user.name("Taylor")
+        end
+      end
+
+      it "0 grouped columns returns { [] => 1 }" do
+        query = UserQuery.new
+
+        query.group_count.should eq({ [] of String => 1 })
+      end
+
+      it "1 grouped columns (age) returns { [42] => 1 }" do
```

Comment:
Could you update these spec/context strings that are really specific to be more generic? This one is incorrect, for example

Reply:
Yea I was thinking about that! Will do